Lib/glyphsLib/builder/anchors.py

```diff
@@ -27,6 +28,10 @@ def to_ufo_glyph_anchors(self, glyph, anchors):
     for anchor in anchors:
         x, y = anchor.position
         anchor_dict = {"name": anchor.name, "x": x, "y": y}
+        if anchor.userData:
+            identifier = str(uuid.uuid4()).upper()
+            anchor_dict["identifier"] = identifier
+            glyph.lib.setdefault("public.objectLibs", {})[identifier] = dict(anchor.userData)
```

Comment:
I think we have a module where we keep all the constants, maybe put public.objectLibs in there